runner/aws_lambda_scripts/deploy_lambda_ecr.sh

```diff
@@ -0,0 +1,173 @@
+# MIT License
+#
+# Copyright (c) 2021 Michal Baczun and EASE lab
```

Comment:
@ustiugov this is also a comment for you.
I think we should change this in the future.
Its not anymore EASE lab.
Also @alannair it seems you created the file so you can add your own name :) 
On the other hand I also found it useless to have names in the licence at all...
Can we not just have `Copyright (c) 2022 The vHive Ecosystem` ?

Reply:
I agree. let's do that: `Copyright (c) 2021 AA, BB and The vHive Ecosystem`